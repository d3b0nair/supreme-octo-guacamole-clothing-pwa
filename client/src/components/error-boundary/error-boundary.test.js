import React from "react";
import { shallow } from "enzyme";
import ErrorBoundary from "./error-boundary.component";

let wrapper;
beforeEach(() => {
  wrapper = shallow(<ErrorBoundary />);
});

describe("ErrorBoundary component", () => {
  it("should mount", () => {
    wrapper.setState({ hasErrored: true });
    expect(wrapper).toMatchSnapshot();
  });
});
