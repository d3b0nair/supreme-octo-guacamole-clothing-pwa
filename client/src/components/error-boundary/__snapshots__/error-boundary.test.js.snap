// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary component should mount 1`] = `
<styled.div>
  <styled.div
    imageUrl="https://i.imgur.com/3suxlvm.png"
  />
  <styled.h2>
    Oh no! We tripped up!
  </styled.h2>
  <styled.h3>
    You had been told you should always check your shoes before getting up from the bleachers. Freshman were known to walk under them and tie peoplesâ€™ shoes together.
  </styled.h3>
  <CustomButton
    inverted={true}
    onClick={[Function]}
  >
    RELOAD PAGE
  </CustomButton>
</styled.div>
`;
